sirepo/package_data/static/js/opal.js

```diff
@@ -223,9 +223,7 @@ SIREPO.app.directive('appFooter', function() {
         },
         template: `
             <div data-common-footer="nav"></div>
-            <div data-import-dialog="" data-title="Import Opal File" data-description="Select an OPAL .in or .madx file." data-file-formats=".in,.madx,.zip">
-              <div data-import-options=""></div>
-            </div>
+            <div data-elegant-import-dialog=""></div>
         `,
```

Comment:
This PR works well, except for the case of uploading a *.in file to opal which requires additional files. It shows "Missing data files" but doesn't provide a way to import the missing files:

![x](https://github.com/radiasoft/sirepo/assets/6516910/1316e519-e8e7-46ca-b446-12792b6c53bc)

To reproduce, open the AWA Photoinjector example, export as zip then edit the Solenoid T7 filenames slightly. Then select "OPAL Source" from the cogwheel and save the AWA-Photoinjector.in file, then edit the *.in file to use the new names for the T7 files. 

Upload the new AWA-Photoinjector.in file, and it should prompt for the T7 files which don't exist on the server.

Reply:
I fixed opal missing data files, but found another edge case. 

When you import ATTR-U-and-W.madx into elegant it hits a key error when the SROTATION elements in madx are converted to SROT. Working on it.